rust/processor/src/db/common/models/account_transaction_models/account_transactions.rs

```diff
@@ -13,13 +13,10 @@ use crate::{
     schema::account_transactions,
     utils::{counters::PROCESSOR_UNKNOWN_TYPE_COUNT, util::standardize_address},
 };
-use ahash::AHashMap;
-use aptos_protos::transaction::v1::{
-    transaction::TxnData, write_set_change::Change, DeleteResource, Event, Transaction,
-    WriteResource,
-};
+use aptos_protos::transaction::v1::{transaction::TxnData, write_set_change::Change, Transaction};
 use field_count::FieldCount;
 use serde::{Deserialize, Serialize};
+use std::collections::HashSet;
```

Comment:
nit: use AHash; my impression is it has better perf.

Reply:
Done